src/db/models/order.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  Field,
+  PrimaryKey,
+  TigrisCollection,
+  TigrisDataTypes,
+} from "@tigrisdata/core";
+import { Product } from "./product";
+
+@TigrisCollection("orders")
+export class Order {
+  @PrimaryKey(TigrisDataTypes.INT64, { order: 1, autoGenerate: true })
+  orderId?: string;
+
+  @Field(TigrisDataTypes.INT64)
+  userId: number;
```

Comment:
This will also be a string because it refers to the userId field in the user model.

Reply:
okay, fixed